Doc/library/email.generator.rst

```diff
@@ -274,9 +274,9 @@ in with information about the part.
 .. rubric:: Footnotes
 
 .. [#] This statement assumes that you use the appropriate setting for
-       ``unixfrom``, and that there are no :mod:`policy` settings calling for
+       ``unixfrom``, and that there are no :mod:`email.policy` settings calling for
```

Comment:
```suggestion
       ``unixfrom``, and that there are no :mod:`~email.policy` settings calling for
```

Reply:
I am not sure that it would be better. No one refer to the module as `policy`. It is known by its full name `email.policy`.

Reply:
The suggestion was so the sentence would read "... there are no policy settings ...", but I think email.policy is fine here.